modules/configuring-cert-based-auth-quay-cloudsql.adoc

```diff
@@ -0,0 +1,195 @@
+:_content-type: PROCEDURE
+[id="configuring-cert-based-auth-quay-sql"]
+= Configuring certificate-based authentication with SQL
+
+The following procedure shows you how to connect {productname} with an SQL database using secure client-side certificates. Using this procedure verifies the SQL server's certificate against a trusted Certificate Authority certificate. This enhances the security of the connection between {productname} and your SQL server, as well as provides easier automation for your deployment. The following example uses Google Cloud Platform's CloudSQL, however it is also applicable for PostgreSQL.
+
+.Prerequisites
+
+* You have generated custom Certificate Authorities (CAs) and your SSL/TLS certificates and keys are available in `PEM` format that will be used to generate an SSL connection with your CloudSQL database. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_quay/{producty}/html-single/securing_red_hat_quay/index#ssl-tls-quay-overview[SSL and TLS for {productname}].
+* You have `base64 decoded` the original config bundle into a `config.yaml` file. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_quay/{producty}/html-single/deploying_the_red_hat_quay_operator_on_openshift_container_platform/index#operator-config-cli-download[Downloading the existing configuration].
+* You are using an externally managed PostgreSQL or CloudSQL database. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_quay/{producty}/html-single/deploying_the_red_hat_quay_operator_on_openshift_container_platform/index#operator-unmanaged-postgres[Using and existing PostgreSQL database] with the `DB_URI` variable set.
+* Your externally managed PostgreSQL or CloudSQL database is configured for SSL/TLS.
+* The `postgres` component of your `QuayRegistry` CRD is set to `managed: false`, and your CloudSQL database is set with the `DB_URI` configuration variable. The following procedure uses `postgresql://<cloudsql_username>:<dbpassword>@34.29.130.58:5432/quay14`.
```

Comment:
34.29.130.58 is myself test instance ip, quay14 is my db, should change to more generic, like: 
`postgresql://<cloudsql_username>:<dbpassword>@<dbhost>:<port>/<database>`.

pls do a search, this also happens in next content

Reply:
Fixed both instances.